localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+d_bloodpaws_adj;Bloodpaw;Sangrepatte;blutpfotisch;;;;;;;;;;;x
+d_snowfalls;Snowfall;Tombeneige;Schneewehen;;Avalancha;;;;;;;;;x
+d_snowfalls_adj;Snowfall;Tombeneige;schneewehisch;;;;;;;;;;;x
+d_burblegobbles;Burblegobble;Brouillegargouille;;;Gorgogluglú;;;;;;;;;x
+d_burblegobbles_adj;Burblegobble;Brouillegargouille;;;;;;;;;;;;x
+d_gurgleboggles;Gurgleboggle;Grouillebarbouille;;;Borbogluglú;;;;;;;;;x
 d_gurgleboggles_adj;Gurgleboggle;;;;;;;;;;;;;x
-d_frenzyhearts;Frenzyheart;;Wildherzen;;;;;;;;;;;x
-d_frenzyhearts_adj;Frenzyheart;;wildherzisch;;;;;;;;;;;x
-d_mistwhispers;Mistwhisper;;Nebelfl�sterer;;;;;;;;;;;x
-d_mistwhispers_adj;Mistwhisper;;nebelfl�sterisch;;;;;;;;;;;x
-d_sparktouched;Sparktouched;;Funkenbespr�hten;;;;;;;;;;;x
-d_sparktouched_adj;Sparktouched;;der Funkenbespr�hten;;;;;;;;;;;x
-d_mosswalkers;Mosswalker;;Mooswandler;;;;;;;;;;;x
-d_mosswalkers_adj;Mosswalker;;mooswandlerisch;;;;;;;;;;;x
-d_winterskorn;Winterskorn;;;;;;;;;;;;;x
-d_winterskorn_adj;Winterskorn;;;;;;;;;;;;;x
+d_frenzyhearts;Frenzyheart;Frénécœur;Wildherzen;;Corazón Frenético;;;;;;;;;x
+d_frenzyhearts_adj;Frenzyheart;Frénécœur;wildherzisch;;;;;;;;;;;x
+d_mistwhispers;Mistwhisper;Murmebrume;Nebelflüsterer;;Susurraneblinas;;;;;;;;;x
+d_mistwhispers_adj;Mistwhisper;Murmebrume;nebelflüsterisch;;;;;;;;;;;x
+d_sparktouched;Sparktouched;Touchétincelles;Funkenbesprühten;;;;;;;;;;;x
+d_sparktouched_adj;Sparktouched;Touchétincelles;der Funkenbesprühten;;;;;;;;;;;x
+d_mosswalkers;Mosswalker;Marchemousses;Mooswandler;;Caminamoho;;;;;;;;;x
+d_mosswalkers_adj;Mosswalker;Marchemousses;mooswandlerisch;;;;;;;;;;;x
+d_winterskorn;Winterskorn;Mörghiver;;;Inbjerskorn;;;;;;;;;x
+d_winterskorn_adj;Winterskorn;Mörghiver;;;;;;;;;;;;x
 d_dragonflayer;Dragonflayer;;Drachenschinder;;;;;;;;;;;x
 d_dragonflayer_adj;Dragonflayer;;drachenschinderisch;;;;;;;;;;;x
 d_kvaldir;Kvaldir;Kvaldir;Kvaldir;;;;;;;;;;;x
 d_kvaldir_adj;Kvaldir;Kvaldir;kvaldirisch;;;;;;;;;;;x
-e_black_empire;Black Empire;Empire noir;Das Schwarze Imperium;;;;;;;;;;;x
-e_black_empire_adj;Black Empire;Noire;schwarz-imperial;;;;;;;;;;;x
+e_black_empire;Black Empire;Empire noir;Schwarze Imperium;;Imperio Negro;;;;;;;;;x
+e_black_empire_adj;Black Empire;Empire Noire;schwarz-imperial;;;;;;;;;;;x
 RED_STEPPES;Red Steppes;;Rote Steppe;;;;;;;;;;;x
 RED_STEPPES_ADJ;Red Steppes;;rot-steppisch;;;;;;;;;;;x
 REDGORGE;Redgorge;;;;;;;;;;;;;x
 REDGORGE_ADJ;Redgorge;;;;;;;;;;;;;x
 k_firelands;Fire Cult;Culte du Feu;Feuerkult;;;;;;;;;;;x
 k_firelands_adj;Fire Cult's;Feu-sectatrice;feuerkultisch;;;;;;;;;;;x
-d_scarlet_crusade;Scarlet Crusade;Croisade �carlate;Scharlachroter Kreuzzug;;;;;;;;;;;x
-d_scarlet_crusade_adj;Scarlet;�carlate;scharlachrot;;;;;;;;;;;x
-d_argent_dawn;Argent Dawn;Aube d'argent;Argentum-D�mmerung;;;;;;;;;;;x
-d_argent_dawn_adj;Argent;Argent�e;argentisch;;;;;;;;;;;x
-d_darkspear;Darkspear;Sombrelance;;;;;;;;;;;;x
+d_scarlet_crusade;Scarlet Crusade;Croisade écarlate;Scharlachrote Kreuzzug;;Cruzada Escarlata;;;;;;;;;x

```

Comment:
I've taken this from the official translation of WoW: https://de.wowhead.com/quest=381/der-scharlachrote-kreuzzug

So up to you, really.

Reply:
As you can see in the link already, they call it "DER scharlachrote Kreuzzug". If you leave the article (der) in place, it's correct, otherwise it has to be "Scharlachroter"

Reply:
Gotcha, will put in next push.